index.js

```diff
@@ -1 +1,25 @@
-console.log("Code your solution!")
+
+const formSubmission = document.querySelector("#user-form");
+
+//listen for submit button
+formSubmission.addEventListener("submit", (event) =>{
+    event.preventDefault();
+    const ul = document.querySelector("ul");
+    const li = document.createElement("li");
+    li.textContent = event.target["listContents"].value;
+    if(li.textContent === ""){
+        const error = document.createElement("p");
+        error.textContent = "List item must contain characters."
+        formSubmission.append(error);
+    }else{
+        ul.append(li);
+    }
+    
+
+    //listen for line through
+
+    li.addEventListener("click", (event) =>{
+        event.target.style.textDecoration = "line-through";
+    })
+
+})
```

Comment:
```suggestion
const formSubmission = document.querySelector("#user-form");

const error = document.createElement("p");  //<--create error message outside of eventListener
error.textContent = "List item must contain characters.";
error.setAttribute("style", "display: none");   //<-- set display:none so that it's hidden by default
formSubmission.append(error);

//listen for submit button
formSubmission.addEventListener("submit", (event) =>{
    event.preventDefault();
    const ul = document.querySelector("ul");
    const li = document.createElement("li");
    li.textContent = event.target["listContents"].value;
    if(li.textContent === ""){
        error.setAttribute("style", "display: block"); //<-- make error message visible when appropriate
    }else{
        error.setAttribute("style", "display: none");  //<-- then hide it again once user enters a valid item
        ul.append(li);
    }
    

    //listen for line through

    li.addEventListener("click", (event) =>{
        event.target.style.textDecoration = "line-through";
    })

})
```
Here's how you can make it so that the error message only appears once. You need to think of it like a **switch** or a **toggle**. Instead of creating and adding a new error message each time, there should only be one error message that is visible when it needs to be, and gets hidden when you don't need it.